about.html

```diff
@@ -0,0 +1,142 @@
+                LAGOS STATE DIGITAL SUMMIT 2022
+            </h1>
+            <p id="about-header-description">
+                The goal of the Lagos Digital Summit 2022 is to bring together all tech enthusiasts, including business owners,
+                freelancers, start-up founders, MDAs, FinTechs, students from inside and beyond the state, corporate bodies, and the
+                general public.
+                
+                This one-day summit will discuss and promote digital technologies and skills for the future, as well as how these
+                abilities might be applied in the near future to solve pressing real-world issues.
+            </p>
+            <p id="about-contact-email">
+                Please contact us via email for further questions about Lagos Digital Summit 2022! 
+                <br>
+                <a href="mailto:fawolemuyiwa@rocketmail.com">fawolemuyiwa@rocketmail.com</a>
+            </p>
+        </div>
+    </header>
+    <section class="container-fluid" id="mission-aim-section">
+        <div class="section-header">
+            <h2 id="main-program-header">Our Mission, Aim and Scope</h2>
+            <span></span>
+        </div>
+        <p id="mission-goal-statement">
+            In order to eliminate uncertainties and change the digital paradigm, we give attention to working on developing
+            technological developments.
+            <br>
+            Our goal is to provide everyone with the fundamental information, proper direction, and mentoring they need to pursue
+            professional insights that address digital requirements.
+        </p>
+    </section>
+    <section id="past-summit-section" class="container-fluid">
+        <div class="section-header">
+            <h2 id="past-summit-header">Past LASG Digital Summits</h2>
+            <span></span>
+        </div>
+        <div class="summit-gallery-container">
+            <div class="summit-gallery" id="gallery1">
+                <div class="gallery-overlay">
+                    <h4>2021</h4>
+                    <p>LASG Summit in Eko Hotel and Suites</p>
+                </div>
+            </div>
+            <div class="summit-gallery" id="gallery2">
+                <div class="gallery-overlay">
+                    <h4>2019</h4>
+                    <p>LASG Summit in Federal Palace Hotel</p>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="container-fluid" id="partners-section">
+        <div class="section-header">
+            <h2 id="partners-header">Partners</h2>
+            <span></span>
+        </div>
+        <div id="partners-logos">
+            <img src="./images/flutterwave.png" alt="daum">
+            <img src="./images/airbnb.png" alt="airbnb">
+            <img src="./images/google.png" alt="google">
+            <img src="./images/daum.png" alt="daum">
+            <img src="./images/capival.png" alt="daum">
+        </div>
+    </section>
```

Comment:
- [ ] Consider making the partner section not displayed on the about page as in Behance design.